source/user-guide/map/index.rst

```diff
@@ -0,0 +1,71 @@
+.. _map-viewer-as-user:
+
+Maps and dataset visualisation
+==============================
+
+The map component in GeoNetwork serves two goals:
+
+- As part of the evaluation of relevance users can visualise a resource discovered in the catalogue in an interactive map viewer to preview the contents.
```

Comment:
The map viewer provides a way for users to interactively assess the relevance of a resource discovered in the catalog